lib/services/devices/deviceService.js

```diff
@@ -27,15 +27,14 @@ var request = require('request'),
     async = require('async'),
     errors = require('../../errors'),
     logger = require('logops'),
+    config = require('../../commonConfig'),
     ngsiParser = require('./../ngsi/ngsiParser'),
     subscriptions = require('../ngsi/subscriptionService'),
     _ = require('underscore'),
     context = {
         op: 'IoTAgentNGSI.DeviceService'
-    },
-    registry,
-    groupRegistry,
-    config;
+    }
+    ;
```

Comment:
I know it is kind of absurd this comment, but maybe you would like to move the `;` to the previous line... :)